contracts/src/v0.8/dev/OptimismSequencerUptimeFeed.sol

```diff
@@ -133,15 +137,16 @@ contract OptimismSequencerUptimeFeed is
    *
    * @param roundId The round ID to record
    * @param status Sequencer status
-   * @param timestamp Block timestamp of status update
+   * @param timestamp The L1 block timestamp of status update
    */
   function recordRound(
     uint80 roundId,
     bool status,
     uint64 timestamp
   ) private {
-    Round memory nextRound = Round(status, timestamp);
-    FeedState memory feedState = FeedState(roundId, status, timestamp);
+    uint64 recordedAt = uint64(block.timestamp);
```

Comment:
Is there a reason why we use the term `recordedAt` for the updatedAt field? I wonder if introducing new terms like this is more confusing

Reply:
just renamed to `updatedAt`